app/models/service_order.rb

```diff
@@ -12,4 +12,18 @@ class ServiceOrder < ActiveRecord::Base
   def assign_user_name
     self.user_name = user.try(:name)
   end
+
+  def self.create_service_order(name, state, requester)
+    ServiceOrder.create(:name => name, :state => state, :user => requester)
```

Comment:
This method does not have any purpose, the callers can use `ServiceOrder.create`.